server/core/src/irods_report_plugins_in_json.cpp

```diff
@@ -48,8 +48,7 @@ namespace irods {
             }
             else {
                 constexpr int sha256_prefix_length = 5;
-                // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds*)
-                plug["checksum_sha256"] = checksum + sha256_prefix_length;
+                plug["checksum_sha256"] = checksum + sha256_prefix_length; // NOLINT: Allow pointer arithmetic
```

Comment:
Please use explicit options for the `NOLINT*` directives.

Without those, we don't know what the reason is behind it.